examples/rbd/pod.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: csirbd-demo-pod
```

Comment:
deployment.yaml in examples/cephfs/deployment.yaml needs changes to address a similar pattern, IOW changing `csicephfs-demo-depl` to `csi-cephfs-demo-depl`

Reply:
Done.